pkg/apis/toolchain/v1alpha1/masteruserrecord_types.go

```diff
@@ -12,6 +12,8 @@ const (
 	MasterUserRecordUserAccountNotReady ConditionType = "UserAccountNotReady"
 	// MasterUserRecordReady means the Master User Record provisioning succeeded
 	MasterUserRecordReady ConditionType = "Ready"
+	// MasterUserRecordProvisioningNotificationCreated is used to track the status of provisioned notification
+	MasterUserRecordProvisioningNotificationCreated ConditionType = "ProvisioningNotificationCreated"
```

Comment:
Since you use `userprovisioned` name for naming the notification template, then I believe that you could rename it accordingly 
```suggestion
	// MasterUserRecordUserProvisionedNotificationCreated means that the Notification CR was created so the user should be notified about the successful provisioning
	MasterUserRecordUserProvisionedNotificationCreated ConditionType = "UserProvisionedNotificationCreated"
```

Reply:
@MatousJobanek sure,sounds good - @alexeykazakov is this okay? I've updated the PR to use this name instead: ca1ba18e26acafbb409d9bfef49d6a33f923d91f